Framework/Algorithms/src/MaxEnt/MaxentTransformMultiFourier.cpp

```diff
@@ -0,0 +1,121 @@
+#include "MantidAlgorithms/MaxEnt/MaxentTransformMultiFourier.h"
+#include <gsl/gsl_fft_complex.h>
+
+namespace Mantid {
+namespace Algorithms {
+
+/** Constructor */
+MaxentTransformMultiFourier::MaxentTransformMultiFourier(
+    MaxentSpaceComplex_sptr dataSpace, MaxentSpace_sptr imageSpace,
+    size_t numSpec)
+    : MaxentTransformFourier(dataSpace, imageSpace), m_numSpec(numSpec),
+      m_linearAdjustments(), m_constAdjustments() {}
+
+/**
+* Transforms a 1D signal from image space to data space, performing an
+* a backward MaxentTransformFourier on it then creating a concatenated
+* copy of the resulting data for each spectrum and applying the adjustments
+* to them.
+* Input is assumed real or complex according to the type of image space
+* given to the constructor.
+* Return value is real or complex according to the type of data space
+* given to the constructor.
+* @param image : [input] Image as a vector
+* @return : The vector in the data space of concatenated spectra
+*/
+std::vector<double>
+MaxentTransformMultiFourier::imageToData(const std::vector<double> &image) {
+
+  std::vector<double> dataOneSpec = MaxentTransformFourier::imageToData(image);
+
+  // Create concatenated copies of transformed data (one for each spectrum)
+  std::vector<double> data;
+  data.reserve(m_numSpec * dataOneSpec.size());
+  for (size_t s = 0; s < m_numSpec; s++) {
+    for (size_t i = 0; i < dataOneSpec.size(); i++) {
+      data.emplace_back(dataOneSpec[i]);
+    }
+  }
+
+  // Apply adjustments (we assume there are sufficient adjustments supplied)
+  double dataR = 0.123456789;
+  double dataI = 0.987654321;
+  if (!m_linearAdjustments.empty() && !m_constAdjustments.empty()) {
+    for (size_t i = 0; i < data.size(); i++) {
```

Comment:
I thought the data was stored as the first N were the real part and the second N are the imaginary part. If that is the case, then I am not convinced by the else statement as the ```dataR``` and ```dataI``` will have the values from the Nth real data

Reply:
This is not the case here. We are dealing with a single (concatenated) spectrum arranged (real1, imaginary1, real2, imaginary2, etc.) as in MaxentTransformFourier.

Reply:
can we have comment stating this?

Reply:
Cooments added in argument descriptions here and in `MaxentTransformFourier.cpp`.